index.html

```diff
@@ -121,6 +146,57 @@ <h2>Project name goes here</h2>
                 </div>
             </div>
         </section>
+        <!-- works desktop -->
+        <section class="grid">
+            <div class="grid-1">
+                
+            <div class="project-grid"></div>
+            <div class="dots"></div>
+            </div>
+            <div class="grid-2">
+                <div class="grid-2-grid-1"></div>
+                <div class="grid-2-grid-2">
+                    <div class="languages-desktop">
+                        <h2>Project name goes here</h2>
+                        <ul class="H-R-J-desktop">
+                            <li class="Html-css">HTML/CSS</li>
+                            <li class="Ruby">RUBY ON RAILS</li>
+                            <li class="Javascript">JAVASCRIPT</li>
+                        </ul>
+                        <button class="see-this-project">
+                            See this project
+                            <img src="images.svg\Union (8).svg" alt="">
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-3"></div>
+            <div class="grid-4"></div>
+            <div class="grid-5">
+            <div class="grid-5-image"></div>
+            <div class="banner">
+                <div class="language-desktop">
+                    <h2>Project name goes ...</h2>
+                    <ul class="H-R-J-desktop">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="plus-one">+1</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (8).svg" alt="">
+                    </button>
+                </div>
+            </div>
+            <div class="grid-5-image-2"></div>  
+            </div>
+            <div class="grid-6"></div>
+            <div class="grid-7"></div>
+            <div class="grid-8"></div>
+            <div class="grid-9"></div>
+            <div class="grid-10"></div>
```

Comment:
- [Optioal] Kindly, those contents are project cards so Please implement the `hover` and the content image is not a background image it is `img` tag because this will be dynamic in the next section when you implement the modal 👌👍
![Screenshot 2023-01-21 175601](https://user-images.githubusercontent.com/52010789/213872837-a4edf9ce-594c-4898-b308-520e8fc0d152.png)